### PR TITLE
Change url project's website

### PR DESCRIPTION
**Pull request by tamplan**
Now the official web site is https://mate-desktop.org/.

### CODE REVIEW

**Review by raveit65 — APPROVED** [review]
LGTM,
thanks.

**Review by selectiveduplicate — CHANGES_REQUESTED** [review]
The change I think wasn't necessary, since the website by default always uses https and the www part doesn't matter.
Besides that, everything seems fine 🙂 , except the following lines in ``gostools.xml`` and ``mate-user-guide.pot``, they should be changed to:

```
Prefix the web page address with https://, as in https://mate-desktop.org.
```

instead of 

```
Prefix the web page address with http://, as in https://mate-desktop.org.
```

File: mate-user-guide/C/gostools.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
     <listitem>
       <para>Enter the command that you want to run in the blank field, or choose from the list of known applications.</para>
       <para>If you enter only the location of a file, an appropriate application will launch to open it.
-      If you enter a web page address, your default web browser will open the page. Prefix the web page address with http://, as in http://www.mate-desktop.org.
+      If you enter a web page address, your default web browser will open the page. Prefix the web page address with http://, as in https://mate-desktop.org.
```
Comment (rbuj) [inline mate-user-guide/C/gostools.xml]: URL prefix should be the same.
```
... address with https://, as in https://mate-desktop.org.
```

File: mate-user-guide/mate-user-guide.pot [hunk 2]
```diff
@@ -13684,7 +13684,7 @@ msgstr ""
 
 #. (itstool) path: listitem/para
 #: C/gostools.xml:50
-msgid "If you enter only the location of a file, an appropriate application will launch to open it. If you enter a web page address, your default web browser will open the page. Prefix the web page address with http://, as in http://www.mate-desktop.org."
+msgid "If you enter only the location of a file, an appropriate application will launch to open it. If you enter a web page address, your default web browser will open the page. Prefix the web page address with http://, as in https://mate-desktop.org."
```
Comment (rbuj) [inline mate-user-guide/mate-user-guide.pot]: URL prefix should be the same.
```
... address with https://, as in https://mate-desktop.org.
```

**Comment by raveit65** [comment]
@tamplan 
Can please follow suggestions for this PR?
So, that we can finish this.
...before opening a bunch of new ones.

Thanks

**Comment by tamplan** [comment]
Ok, i read and modify the files.

**Comment by tamplan** [comment]
Done.

**Comment by raveit65** [comment]
Thanks for update, i will squash both commits into one :)

**Comment by raveit65** [comment]
merged
https://github.com/mate-desktop/mate-user-guide/commit/7ac840ab3a8f7c0cae66784305242db0b5793fea
Thank you.

**Comment by tamplan** [comment]
> Thanks for update, i will squash both commits into one :)

Tanks for squashing :)

**Pull request closed without merging**